sky/server/common.py

```diff
@@ -170,7 +170,7 @@ def start_uvicorn_in_background(deploy: bool = False, host: str = '127.0.0.1'):
         api_server_cmd += ' --deploy'
     if host is not None:
         api_server_cmd += f' --host {host}'
-    cmd = f'{sys.executable} {api_server_cmd} > {log_path} 2>&1'
+    cmd = f'{sys.executable} {api_server_cmd} > {log_path} 2>&1 < /dev/null'
```

Comment:
Just a comment: We have turned off stdin for many underlying `command_runner.run` function's stdin. Supposedly, there should be no underlying process taking the stdin, but there might be some unhandled ones. Adding this should be safer.